cerbero/bootstrap/build_tools.py

```diff
@@ -120,8 +124,36 @@ def _setup_env(self):
         self.recipes = self.BUILD_TOOLS
         self.recipes += self.PLAT_BUILD_TOOLS.get(self.config.platform, [])
 
+    def insert_python_site(self):
+        try:
+            import setuptools.version as stv
+        except ImportError:
+            return
+
+        version = [int(v) for v in stv.__version__.split('.')]
+        if len(version) < 1 or version[:1] < [49]:
+            return
+
+        m.warning('detected setuptools >= 49.0.0, installing fallback site.py file. '
+            'See https://github.com/pypa/setuptools/issues/2295')
+
+        # Since python-setuptools 49.0.0, site.py is not installed by
+        # easy_install/setup.py anymore which breaks python installs outside
+        # the system prefix.
+        # https://github.com/pypa/setuptools/issues/2295
+        #
+        # Install the previously installed site.py ourselves as a workaround
+        config = self.cookbook.get_config()
+
+        py_prefix = sysconfig.get_path('purelib', 'posix_prefix', vars={'base': ''})
+        # Must strip \/ to ensure that the path is relative
+        py_prefix = PurePath(config.prefix) / PurePath(py_prefix.strip('\\/'))
+        src_file = os.path.join(os.path.dirname(__file__), 'site-patch.py')
+        shutil.copy(src_file, py_prefix / 'site.py')        
+        
```

Comment:
Extra spaces here

Reply:
Also included in the community :(
I only did a cherry-pick